shared/fs/popups/row-action-popup-container.js

```diff
@@ -159,14 +167,24 @@ const mergeProps = (stateProps, dispatchProps) => {
     lastWriter: pathItem.lastWriter.username,
     name: pathItem.name,
     size: pathItem.size,
+    needLoadMimeType: pathItem.type === 'file' && pathItem.mimeType === '',
     childrenFolders,
     childrenFiles,
     itemStyles,
     menuItems,
     onHidden,
+    loadMimeType: () => _loadMimeType(path),
   }
 }
 
-export default compose(connect(mapStateToProps, mapDispatchToProps, mergeProps), setDisplayName('Popup'))(
-  Popup
-)
+export default compose(
+  connect(mapStateToProps, mapDispatchToProps, mergeProps),
+  lifecycle({
+    componentDidMount() {
+      if (this.props.needLoadMimeType) {
+        this.props.loadMimeType()
+      }
+    },
+  }),
+  setDisplayName('Popup')
```

Comment:
probably want to move this `setDisplayName` up

Reply:
Ah right! I forgot to fix this.